package-rust.py

```diff
@@ -176,15 +180,26 @@ def run(args):
 if channel == "stable":
     CFG_RELEASE=CFG_RELEASE_NUM
     CFG_PACKAGE_VERS=CFG_RELEASE_NUM
+    # UpgradeCode shoud stay the same for all MSI versions in channel
+    CFG_UPGRADE_CODE="1C7CADA5-D117-43F8-A356-DF15F9FBEFF6"
```

Comment:
> leaving PATH alone (or at least making this feature not selected by default

it's not selected by default write now. It has `Level="5"` which is higher than the default [INSTALLLEVEL](http://msdn.microsoft.com/en-us/library/aa369536%28v=vs.85%29.aspx)